### DISCUSSION TITLE
Camera Offset?

### DISCUSSION THREAD

**Discussion by DioIsntFunni** [Leather128/LeathersFunkinEngine#109] [Q&A] [answered]
Is there anyway to change the Default position of the camera using Modcharts or in the stage's Json Or is this just not added yet

**Comment by Leather128** [comment, marked as answer]
there is a way of offseting the camera for each character when they're on screen its the `cameraOffset`

and it's used in character config.jsons like this:

```json
"cameraOffset": [50, 50]
```

although keep in mind that a default camera offset is always applied, and some hardcoded characters (from base game weeks) can't have their camera offsets changed.
